static/js/zamboni/devhub.js

```diff
@@ -691,6 +691,91 @@ function initVersions() {
             $('.current-version-warning', this).toggle(is_current);
             return true;
         }});
+
+    function loadReviewHistory(div, nextLoad) {
+        div.removeClass("hidden");
+        div.prop("style", "");
```

Comment:
I don't understand why you need to play with `style` here and in all other places you do it below ?